src/cowboy_http2.erl

```diff
@@ -666,6 +661,24 @@ maybe_send_data(State=#state{http2_machine=HTTP2Machine0}, StreamID, IsFin, Data
 			send_data(State#state{http2_machine=HTTP2Machine}, SendData)
 	end.
 
+cal_and_update_window(Transport, Socket, HTTP2Machine) ->
```

Comment:
Just name them `update_window`.

Reply:
But we already have a update_window, right?